national-natural-science-foundation-of-china-2017.csl

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="zh-CN">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>National Natural Science Foundation of China 2017</title>
+    <title-short>NSFC_2017</title-short>
+    <id>http://www.zotero.org/styles/national-natural-science-foundation-of-china-2017</id>
+    <link href="http://www.zotero.org/styles/national-natural-science-foundation-of-china-2017" rel="self"/>
+    <link href="http://gradschool.ustc.edu.cn/ylb/material/xw/wdxz/19.pdf" rel="documentation"/>
+    <author>
+      <name>Jiongdd</name>
+      <email>wangjiang07@foxmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <category field="science"/>
+    <category field="generic-base"/>
+    <summary>This style just partly implemented what the National Natural Science Foundation of China requires.</summary>
```

Comment:
> This style just partly implemented what the National Natural Science Foundation of China requires.

@jiongdd, can you explain a little what you mean by this? Did you have trouble getting the style right, or did you only check it for a low number of different item types (like articles, books, book chapters, etc.)?